Token.js

```diff
@@ -664,13 +666,22 @@ class Token {
 				old.find("img").removeClass("token-round");
 			}
 			
-			if(this.options.locked){
+			if(this.options.locked && !this.options.restrictPlayerMove){
```

Comment:
also here
if (this.options.locked  ||   (!window.DM && this.options.restrictPlayerMove)